Tree/Binary_tree.py

```diff
@@ -0,0 +1,29 @@
+class BinaryTree:
+    def __init__(self, data) -> None:
+        self.data = data
+        self.leftChild = None
```

Comment:
But for variable name it has become standard to use snake_case. Lets start doing that.
```suggestion
        self.left_child = None
```

Reply:
From next time.